brush-parser/src/lib.rs

```diff
@@ -11,10 +11,12 @@ pub mod word;
 
 mod error;
 mod parser;
+mod parser2;
 mod tokenizer;
 
 pub use error::{ParseError, TestCommandParseError, WordParseError};
 pub use parser::{parse_tokens, Parser, ParserOptions, SourceInfo};
+pub use parser2::Parser as WinnowParser;
```

Comment:
Is there a reason not to just name it `WinnowParser` in the first place? It would be great to get a more descriptive and understandable name than `parser2` when it gets checked in.